cmd/traffic_cache_tool/CacheDefs.h

```diff
@@ -488,6 +349,76 @@ constexpr int CACHE_BLOCK_SIZE          = (1 << CACHE_BLOCK_SHIFT); // 512, smal
 
 namespace ct
 {
+#define dir_big(_e) ((uint32_t)((((_e)->w[1]) >> 8) & 0x3))
```

Comment:
We need to

* Make all of these `inline`
* Put them in a header that can be included by Cache Tool without linking issues.

OTSOTPR.